modules/tools/powerControl.lua

```diff
@@ -1,19 +1,42 @@
 local component = require("component")
-
+local serialization = require("serialization")
 local powerControl = {}
 
+local powerControlData = {}
+
 local redstone = nil
+local enableLevel = 0
+local disableLevel = 0
 
-local function getPercentage(data)
-    return data.storedEU / data.EUCapacity
+
+local function save(data)
+    local file = io.open("/home/NIDAS/settings/powerControlData", "w")
+    enableLevel = tonumber(powerControlData.enableLevel)
+    disableLevel = tonumber(powerControlData.disableLevel)
+    if enableLevel ~= nil and disableLevel ~= nil then
+        file:write(serialization.serialize(powerControlData))
+        file:close()
+    end
+end
```

Comment:
Are these `load()` and `save(data)` functions repeated in the same way?
If so, move them to a `utils` function

Reply:
They all have special handling depending on the module they work with.

It could be done but would require a more unified structure for everything, which is a fairly major rewrite.